R/slide.R

```diff
@@ -573,81 +663,11 @@ epi_slide_mean = function(x, col_name, ..., before, after, ref_time_values,
     after <- 0L
   }
 
-  pad_early_dates <- c()
```

Comment:
This chunk is turned into a fn below so that `epi_slide` can also use it to get a complete date sequence using the complicated `time_step` handling.